adx/resource_adx_function.go

```diff
@@ -0,0 +1,117 @@
+package adx
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	"github.com/Azure/azure-kusto-go/kusto"
+	"github.com/Azure/azure-kusto-go/kusto/unsafe"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type ADXFunctionResult struct {
+	Name string
+	Parameters string
+	Body string
+}
+
+func resourceADXFunction() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceADXFunctionCreate,
+		UpdateContext: resourceADXFunctionCreate,
```

Comment:
At least by azurerm convention - if we use the same func to do both Create and Update - func name is resourceXXXCreateUpdate. Could we amend that please?

Reply:
fixed